pwiz_tools/Skyline/Controls/Graphs/DetectionToolbarProperties.resx

```diff
@@ -120,4 +120,595 @@
+    <value>groupBox2</value>
+  </data>
+  <data name="&gt;&gt;groupBox2.Type" xml:space="preserve">
+    <value>System.Windows.Forms.GroupBox, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </data>
+  <data name="&gt;&gt;groupBox2.Parent" xml:space="preserve">
+    <value>$this</value>
+  </data>
+  <data name="&gt;&gt;groupBox2.ZOrder" xml:space="preserve">
+    <value>1</value>
+  </data>
+  <data name="tbAtLeastN.Location" type="System.Drawing.Point, System.Drawing">
+    <value>11, 21</value>
+  </data>
+  <data name="tbAtLeastN.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>3, 2, 3, 2</value>
+  </data>
+  <data name="tbAtLeastN.Size" type="System.Drawing.Size, System.Drawing">
+    <value>203, 56</value>
+  </data>
+  <data name="tbAtLeastN.TabIndex" type="System.Int32, mscorlib">
+    <value>0</value>
+  </data>
+  <data name="&gt;&gt;tbAtLeastN.Name" xml:space="preserve">
+    <value>tbAtLeastN</value>
+  </data>
+  <data name="&gt;&gt;tbAtLeastN.Type" xml:space="preserve">
+    <value>System.Windows.Forms.TrackBar, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </data>
+  <data name="&gt;&gt;tbAtLeastN.Parent" xml:space="preserve">
+    <value>gbAtLeastN</value>
+  </data>
+  <data name="&gt;&gt;tbAtLeastN.ZOrder" xml:space="preserve">
+    <value>0</value>
+  </data>
+  <data name="gbAtLeastN.Location" type="System.Drawing.Point, System.Drawing">
+    <value>220, 210</value>
+  </data>
+  <data name="gbAtLeastN.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>3, 2, 3, 2</value>
+  </data>
+  <data name="gbAtLeastN.Padding" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>3, 2, 3, 2</value>
+  </data>
+  <data name="gbAtLeastN.Size" type="System.Drawing.Size, System.Drawing">
+    <value>220, 79</value>
+  </data>
+  <data name="gbAtLeastN.TabIndex" type="System.Int32, mscorlib">
+    <value>8</value>
+  </data>
+  <data name="gbAtLeastN.Text" xml:space="preserve">
+    <value>At least N &amp;replicates</value>
+  </data>
+  <data name="&gt;&gt;gbAtLeastN.Name" xml:space="preserve">
+    <value>gbAtLeastN</value>
+  </data>
+  <data name="&gt;&gt;gbAtLeastN.Type" xml:space="preserve">
+    <value>System.Windows.Forms.GroupBox, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </data>
+  <data name="&gt;&gt;gbAtLeastN.Parent" xml:space="preserve">
+    <value>$this</value>
+  </data>
+  <data name="&gt;&gt;gbAtLeastN.ZOrder" xml:space="preserve">
+    <value>0</value>
+  </data>
+  <metadata name="$this.Localizable" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>True</value>
+  </metadata>
+  <data name="$this.AutoScaleDimensions" type="System.Drawing.SizeF, System.Drawing">
+    <value>8, 16</value>

```

Comment:
This AutoScaleDimensions proves that you are not using 100% on the monitor you used for the form designer.